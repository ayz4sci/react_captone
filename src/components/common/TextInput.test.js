import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import TextInput from './TextInput';

it('renders correctly', () => {
    const props = { 
        name: "Lol",
        label: "lol",
        type: "text",
        onChange: jest.fn()
    };

    const wrapper = shallow(<TextInput {...props} />, {mode: 'shallow'})
    expect(toJson(wrapper)).toMatchSnapshot();
});