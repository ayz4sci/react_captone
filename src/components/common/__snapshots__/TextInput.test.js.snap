// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="form-group"
>
  <label
    htmlFor="Lol"
  >
    lol
  </label>
  <div
    className="field"
  >
    <input
      className="form-control"
      name="Lol"
      onChange={[MockFunction]}
      type="text"
    />
  </div>
</div>
`;
